### ISSUE TITLE
All spawn request blocked while player reconnect to a server

### ISSUE THREAD

**Issue by DPurple** [MarkusBordihn/adaptive_performance_tweaks#22]
**Describe the bug**
While player disconnect and reconnect to server, aptweaks failed to track this player's location, causing all mob spawn requests by this player was blocked.

**To Reproduce**
1.join the server, wait for just few seconds and let aptweaks track you.
2.leave the server, and re-join
3.move away or teleport to another dimension
4.you will found no more mobs spawned nearby you

**Expected behavior**
aptweaks should handle player reconnect.

**Screenshots**
No need

**Debugging**
Before reporting a issue please use `/aptweaks debug true` to enable debugging and check if there is anything relevant in the debug.log file.

**Is the issue happening in combination with other mods? (If yes, please provide a full list)**
not needed, i think...

**Additional context**
there's a pull request to fix this

**Comment by DPurple** [comment]
Unfortunately problem still exists.
My commit didn't hit the target.
It does solve the re-connect problem, but there was more comming.
I don't know which mod is touching player entities but i can found those pairs in debug.log
[22:55:27.178] [Server thread/DEBUG] [APTweaks:EntityManager/]: Player ServerPlayerEntity['XXXX'/968, l='ServerLevel[world]', x=-1569.70, y=68.00, z=6787.70] leaved world.
[22:55:27.179] [Server thread/DEBUG] [APTweaks:EntityManager/]: Player ServerPlayerEntity['XXXX'/968, l='ServerLevel[world]', x=-1569.70, y=68.00, z=6787.70] joined world.
Yes, accurately, this is not APTweaks's fault. 
About that commit, I misread line 68 in PlayerPositionManager.java.
I thought PlayerManager.getPlayerList is something linke MinecraftServer.getPlayerList().getPlayers() stuff.
But it was a copy of playerlist managed by aptweak itself which using forge events to track players.
I don't think it's a good idea.
So I don't wanna make a mess of your code, but currently changing PlayerManger.getPlayerList()
into
ServerLifecycleHooks.getCurrentServer().getPlayerList().getPlayers()
fixed my problem.

I'm sorry if I troubled you.

**Comment by MarkusBordihn** [comment]
No worries. Thanks a lot for input and I think I figured out the issue.

The PlayerManager only add a player when is is verified that the player is connected and able to move around.
This test has an automatic timeout, which means that it could take up to 90secs until a player is considered "available" and his position is tracked and updated.

I changed this behavior so that the player position is tracked independently of this player validation.
